tests/e2e/specs/activate-and-setup/complete-industry-section.js

```diff
@@ -17,28 +22,29 @@ export async function completeIndustrySection( expectedIndustryCount = 8 ) {
 		expectedIndustryCount
 	);
 
-	const industryCheckboxes = await page.$$(
-		'.components-checkbox-control__input'
+	// Select just "fashion" and "health/beauty" to get the single checkbox business section when
+	// filling out details for a US store.
+	const fashionLabel = await getElementByText(
+		'label',
+		'Fashion, apparel, and accessories'
 	);
+	// get sibling checkbox
+	const fashionCheckboxContainer = await getPreviousSibling( fashionLabel );
+	const fashionCheckbox = await fashionCheckboxContainer.$( 'input' );
 
-	const industryLabels = await page.$$(
-		'.components-checkbox-control__label'
-	);
-
-	// Select all industries except for CBD to fulfill conditions required by business section tests.
-	for ( let i = 0; i < expectedIndustryCount; i++ ) {
```

Comment:
Before we just filled out all the check boxes, I've introduced some helpers to make it easier to fill out a specified checkbox which allows us to control which segmentation happens.